code/src/Components/Result.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export const Result = ({ name, mood, drink }) => {
+  return (
+    <div className="resultComponent">
+      <h2>Thank you for answering, this is how you responded:</h2>
+      <div className="results">
+        <p>Name: {name}</p>
+        <p>Current mood: {mood}</p>
+        <p>I prefer to drink: {drink}</p>
+      </div>
+      <h3>PS. You are doing great! You should be proud of yourself.</h3>
```

Comment:
The semantic structure has throughout the site been great! But I wonder if it would be better here to use a bold p instead of an h3, since the use of the h3 sort of implies that something will follow?